### ISSUE TITLE
JNI DETECTED ERROR IN APPLICATION: JNI FindClass called with pending exception java.lang.ClassNotFoundException: Didn't find class "net.sqlcipher.database.SQLiteDatabase"

### ISSUE THREAD

**Issue by nikz-dev** [sqlcipher/android-database-sqlcipher#337]
On Android with the latest version of SQLCipher (3.5.7), clone sqlcipher from https://github.com/sqlcipher/android-database-sqlcipher.git and build so files, now while using libsqlcipher.so in sample project getting java.lang.ClassNotFoundException
This is the full stacktrace:

```
05-11 10:53:30.384 22387-22403/? E/art: Thread attaching while runtime is shutting down: Binder_1
05-11 10:53:31.299 779-891/? E/native: do suspend false
05-11 10:53:31.351 2303-2307/? E/ANDR-PERF-LOCK: Failed to reset optimization for resource: 4 level: 0
05-11 10:53:31.423 1434-22289/? E/ctxmgr: [ProducerActiveIntervalImpl]addNewAppInterval: already contained:appKey=cdv@71c1b10a,appInterval=cdt@14cddfb,record=cjv{key=cjx{com.google.android.googlequicksearchbox, 10026, account#-248808883#, com.google.android.apps.gsa.staticplugins.proactive.GmsContextManagerClientHelper, 3708e5fc-2e68-458a-b194-60435c64681a}, t=1494477952131, name=1, lifetime=2, production=null, retention=null, dispatch=1, consumer=cfn@32dd3462-213796867}
05-11 10:53:31.639 1434-22290/? E/ctxmgr: [AppIntervalImpl]closeInterval: ongoing
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410] JNI DETECTED ERROR IN APPLICATION: JNI FindClass called with pending exception java.lang.ClassNotFoundException: Didn't find class "net.sqlcipher.database.SQLiteDatabase" on path: DexPathList[[zip file "/data/app/x.xx.samplejni-1/base.apk"],nativeLibraryDirectories=[/data/app/x.xx.samplejni-1/lib/arm, /data/app/x.xx.samplejni-1/base.apk!/lib/armeabi, /vendor/lib, /system/lib]]
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:511)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:469)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.String java.lang.Runtime.nativeLoad(java.lang.String, java.lang.ClassLoader, java.lang.String) (Runtime.java:-2)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.String java.lang.Runtime.doLoad(java.lang.String, java.lang.ClassLoader) (Runtime.java:435)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void java.lang.Runtime.loadLibrary(java.lang.String, java.lang.ClassLoader) (Runtime.java:370)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void java.lang.System.loadLibrary(java.lang.String) (System.java:1076)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void x.xx.samplejni.MainActivity.<clinit>() (MainActivity.java:23)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Object java.lang.Class.newInstance!() (Class.java:-2)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.Activity android.app.Instrumentation.newActivity(java.lang.ClassLoader, java.lang.String, android.content.Intent) (Instrumentation.java:1067)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2317)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2476)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void android.app.ActivityThread.-wrap11(android.app.ActivityThread, android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:-1)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1344)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void android.os.Looper.loop() (Looper.java:148)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:5417)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:726)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:616)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410] 
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]     in call to FindClass
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]     from java.lang.String java.lang.Runtime.nativeLoad(java.lang.String, java.lang.ClassLoader, java.lang.String)
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410] "main" prio=5 tid=1 Runnable
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   | group="main" sCount=0 dsCount=0 obj=0x7318a2a0 self=0xb4db6500
05-11 10:53:32.028 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   | sysTid=22440 nice=0 cgrp=default sched=0/0 handle=0xb6faeb34
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   | state=R schedstat=( 53725266 34871818 121 ) utm=4 stm=1 core=1 HZ=100
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   | stack=0xbe0f5000-0xbe0f7000 stackSize=8MB
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   | held mutexes= "mutator lock"(shared held)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #00 pc 00370e01  /system/lib/libart.so (_ZN3art15DumpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiPKcPNS_9ArtMethodEPv+160)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #01 pc 0035046f  /system/lib/libart.so (_ZNK3art6Thread4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+150)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #02 pc 0025a725  /system/lib/libart.so (_ZN3art9JavaVMExt8JniAbortEPKcS2_+740)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #03 pc 0025adfd  /system/lib/libart.so (_ZN3art9JavaVMExt9JniAbortVEPKcS2_St9__va_list+64)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #04 pc 000fd1d1  /system/lib/libart.so (_ZN3art11ScopedCheck6AbortFEPKcz+32)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #05 pc 001022e5  /system/lib/libart.so (_ZN3art11ScopedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE.constprop.95+5072)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #06 pc 00104493  /system/lib/libart.so (_ZN3art8CheckJNI9FindClassEP7_JNIEnvPKc+386)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #07 pc 000e4f97  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so (???)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #08 pc 000e572b  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so (???)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #09 pc 0025b387  /system/lib/libart.so (_ZN3art9JavaVMExt17LoadNativeLibraryEP7_JNIEnvRKNSt3__112basic_stringIcNS3_11char_traitsIcEENS3_9allocatorIcEEEEP8_jobjectPS9_+1238)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #10 pc 002d13d7  /system/lib/libart.so (_ZN3artL18Runtime_nativeLoadEP7_JNIEnvP7_jclassP8_jstringP8_jobjectS5_+194)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   native: #11 pc 00212105  /data/dalvik-cache/arm/system@framework@boot.oat (Java_java_lang_Runtime_nativeLoad__Ljava_lang_String_2Ljava_lang_ClassLoader_2Ljava_lang_String_2+144)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Runtime.nativeLoad(Native method)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Runtime.doLoad(Runtime.java:435)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   - locked <0x0947938f> (a java.lang.Runtime)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Runtime.loadLibrary(Runtime.java:370)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.System.loadLibrary(System.java:1076)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at x.xx.samplejni.MainActivity.<clinit>(MainActivity.java:23)
05-11 10:53:32.029 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.Class.newInstance!(Native method)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.Instrumentation.newActivity(Instrumentation.java:1067)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2317)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.ActivityThread.-wrap11(ActivityThread.java:-1)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.os.Handler.dispatchMessage(Handler.java:102)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.os.Looper.loop(Looper.java:148)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at android.app.ActivityThread.main(ActivityThread.java:5417)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at java.lang.reflect.Method.invoke!(Native method)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
05-11 10:53:32.031 22440-22440/x.xx.samplejni A/art: art/runtime/java_vm_ext.cc:410] 
05-11 10:53:32.149 1923-2050/? E/BrcmNfcJni: nfaConnectionCallback: unknown event ????
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] Runtime aborting...
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] Aborting thread:
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "main" prio=5 tid=1 Native
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=0 dsCount=0 obj=0x7318a2a0 self=0xb4db6500
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22440 nice=0 cgrp=default sched=0/0 handle=0xb6faeb34
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=R schedstat=( 70161467 49269159 286 ) utm=5 stm=2 core=1 HZ=100
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xbe0f5000-0xbe0f7000 stackSize=8MB
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes= "abort lock"
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 00370e01  /system/lib/libart.so (_ZN3art15DumpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiPKcPNS_9ArtMethodEPv+160)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 0035046f  /system/lib/libart.so (_ZNK3art6Thread4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+150)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 00333827  /system/lib/libart.so (_ZNK3art10AbortState10DumpThreadERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEPNS_6ThreadE+26)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 00333abf  /system/lib/libart.so (_ZN3art7Runtime5AbortEv+562)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 000f45fb  /system/lib/libart.so (_ZN3art10LogMessageD2Ev+2226)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 0025aa4f  /system/lib/libart.so (_ZN3art9JavaVMExt8JniAbortEPKcS2_+1550)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #06 pc 0025adfd  /system/lib/libart.so (_ZN3art9JavaVMExt9JniAbortVEPKcS2_St9__va_list+64)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #07 pc 000fd1d1  /system/lib/libart.so (_ZN3art11ScopedCheck6AbortFEPKcz+32)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #08 pc 001022e5  /system/lib/libart.so (_ZN3art11ScopedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE.constprop.95+5072)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #09 pc 00104493  /system/lib/libart.so (_ZN3art8CheckJNI9FindClassEP7_JNIEnvPKc+386)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #10 pc 000e4f97  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so (???)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #11 pc 000e572b  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so (???)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #12 pc 0025b387  /system/lib/libart.so (_ZN3art9JavaVMExt17LoadNativeLibraryEP7_JNIEnvRKNSt3__112basic_stringIcNS3_11char_traitsIcEENS3_9allocatorIcEEEEP8_jobjectPS9_+1238)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #13 pc 002d13d7  /system/lib/libart.so (_ZN3artL18Runtime_nativeLoadEP7_JNIEnvP7_jclassP8_jstringP8_jobjectS5_+194)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #14 pc 00212105  /data/dalvik-cache/arm/system@framework@boot.oat (???)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Runtime.nativeLoad(Native method)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Runtime.doLoad(Runtime.java:435)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - locked <0x0947938f> (a java.lang.Runtime)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Runtime.loadLibrary(Runtime.java:370)
05-11 10:53:32.189 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.System.loadLibrary(System.java:1076)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at x.xx.samplejni.MainActivity.<clinit>(MainActivity.java:23)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Class.newInstance!(Native method)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.Instrumentation.newActivity(Instrumentation.java:1067)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2317)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.-wrap11(ActivityThread.java:-1)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.os.Handler.dispatchMessage(Handler.java:102)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.os.Looper.loop(Looper.java:148)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.main(ActivityThread.java:5417)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.reflect.Method.invoke!(Native method)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] Pending exception java.lang.ClassNotFoundException: Didn't find class "net.sqlcipher.database.SQLiteDatabase" on path: DexPathList[[zip file "/data/app/x.xx.samplejni-1/base.apk"],nativeLibraryDirectories=[/data/app/x.xx.samplejni-1/lib/arm, /data/app/x.xx.samplejni-1/base.apk!/lib/armeabi, /vendor/lib, /system/lib]]
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:511)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:469)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.String java.lang.Runtime.nativeLoad(java.lang.String, java.lang.ClassLoader, java.lang.String) (Runtime.java:-2)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.String java.lang.Runtime.doLoad(java.lang.String, java.lang.ClassLoader) (Runtime.java:435)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void java.lang.Runtime.loadLibrary(java.lang.String, java.lang.ClassLoader) (Runtime.java:370)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void java.lang.System.loadLibrary(java.lang.String) (System.java:1076)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void x.xx.samplejni.MainActivity.<clinit>() (MainActivity.java:23)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Object java.lang.Class.newInstance!() (Class.java:-2)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.Activity android.app.Instrumentation.newActivity(java.lang.ClassLoader, java.lang.String, android.content.Intent) (Instrumentation.java:1067)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2317)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2476)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void android.app.ActivityThread.-wrap11(android.app.ActivityThread, android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:-1)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1344)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void android.os.Looper.loop() (Looper.java:148)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:5417)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:726)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:616)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] Dumping all threads without appropriate locks held: thread list lock mutator lock
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] All threads:
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] DALVIK THREADS (9):
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "main" prio=5 tid=1 Runnable
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=0 dsCount=0 obj=0x7318a2a0 self=0xb4db6500
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22440 nice=0 cgrp=default sched=0/0 handle=0xb6faeb34
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=R schedstat=( 83265476 65263847 309 ) utm=5 stm=3 core=2 HZ=100
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xbe0f5000-0xbe0f7000 stackSize=8MB
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes= "abort lock" "mutator lock"(shared held)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 00370e01  /system/lib/libart.so (_ZN3art15DumpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiPKcPNS_9ArtMethodEPv+160)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 0035046f  /system/lib/libart.so (_ZNK3art6Thread4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+150)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 0035a373  /system/lib/libart.so (_ZN3art14DumpCheckpoint3RunEPNS_6ThreadE+442)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 0035af31  /system/lib/libart.so (_ZN3art10ThreadList13RunCheckpointEPNS_7ClosureE+212)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 0035b45f  /system/lib/libart.so (_ZN3art10ThreadList4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+142)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 00333a35  /system/lib/libart.so (_ZN3art7Runtime5AbortEv+424)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #06 pc 000f45fb  /system/lib/libart.so (_ZN3art10LogMessageD2Ev+2226)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #07 pc 0025aa4f  /system/lib/libart.so (_ZN3art9JavaVMExt8JniAbortEPKcS2_+1550)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #08 pc 0025adfd  /system/lib/libart.so (_ZN3art9JavaVMExt9JniAbortVEPKcS2_St9__va_list+64)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #09 pc 000fd1d1  /system/lib/libart.so (_ZN3art11ScopedCheck6AbortFEPKcz+32)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #10 pc 001022e5  /system/lib/libart.so (_ZN3art11ScopedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE.constprop.95+5072)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #11 pc 00104493  /system/lib/libart.so (_ZN3art8CheckJNI9FindClassEP7_JNIEnvPKc+386)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #12 pc 000e4f97  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so (???)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #13 pc 000e572b  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so (???)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #14 pc 0025b387  /system/lib/libart.so (_ZN3art9JavaVMExt17LoadNativeLibraryEP7_JNIEnvRKNSt3__112basic_stringIcNS3_11char_traitsIcEENS3_9allocatorIcEEEEP8_jobjectPS9_+1238)
05-11 10:53:32.190 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #15 pc 002d13d7  /system/lib/libart.so (_ZN3artL18Runtime_nativeLoadEP7_JNIEnvP7_jclassP8_jstringP8_jobjectS5_+194)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #16 pc 00212105  /data/dalvik-cache/arm/system@framework@boot.oat (Java_java_lang_Runtime_nativeLoad__Ljava_lang_String_2Ljava_lang_ClassLoader_2Ljava_lang_String_2+144)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Runtime.nativeLoad(Native method)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Runtime.doLoad(Runtime.java:435)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - locked <0x0947938f> (a java.lang.Runtime)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Runtime.loadLibrary(Runtime.java:370)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.System.loadLibrary(System.java:1076)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at x.xx.samplejni.MainActivity.<clinit>(MainActivity.java:23)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Class.newInstance!(Native method)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.Instrumentation.newActivity(Instrumentation.java:1067)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2317)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.-wrap11(ActivityThread.java:-1)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.os.Handler.dispatchMessage(Handler.java:102)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.os.Looper.loop(Looper.java:148)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at android.app.ActivityThread.main(ActivityThread.java:5417)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.reflect.Method.invoke!(Native method)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "Signal Catcher" prio=5 tid=2 WaitingInMainSignalCatcherLoop
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d5e0a0 self=0xaee52000
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22445 nice=0 cgrp=default sched=0/0 handle=0xb446d930
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 1409583 26146 4 ) utm=0 stm=0 core=0 HZ=100
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb4371000-0xb4373000 stackSize=1014KB
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 00040b68  /system/lib/libc.so (__rt_sigtimedwait+12)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 0001c963  /system/lib/libc.so (sigwait+22)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 0033a725  /system/lib/libart.so (_ZN3art13SignalCatcher13WaitForSignalEPNS_6ThreadERNS_9SignalSetE+76)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 0033b887  /system/lib/libart.so (_ZN3art13SignalCatcher3RunEPv+218)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 0003f45f  /system/lib/libc.so (_ZL15__pthread_startPv+30)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 00019b43  /system/lib/libc.so (__start_thread+6)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   (no managed stack frames)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "JDWP" prio=5 tid=3 WaitingInMainDebuggerLoop
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d610a0 self=0xacc76d00
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22446 nice=0 cgrp=default sched=0/0 handle=0xb436e930
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 2093490 903385 11 ) utm=0 stm=0 core=0 HZ=100
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb4272000-0xb4274000 stackSize=1014KB
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 00040a84  /system/lib/libc.so (__pselect6+20)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 0001bcb5  /system/lib/libc.so (select+60)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 004012d3  /system/lib/libart.so (_ZN3art4JDWP12JdwpAdbState15ProcessIncomingEv+218)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 00266e47  /system/lib/libart.so (_ZN3art4JDWP9JdwpState3RunEv+314)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 00267cc5  /system/lib/libart.so (_ZN3art4JDWPL15StartJdwpThreadEPv+16)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 0003f45f  /system/lib/libc.so (_ZL15__pthread_startPv+30)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #06 pc 00019b43  /system/lib/libc.so (__start_thread+6)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   (no managed stack frames)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "ReferenceQueueDaemon" prio=5 tid=4 Waiting
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d5b640 self=0xacc79000
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22447 nice=0 cgrp=default sched=0/0 handle=0xb426f930
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 611459 144270 10 ) utm=0 stm=0 core=3 HZ=100
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb416d000-0xb416f000 stackSize=1038KB
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 000175e4  /system/lib/libc.so (syscall+28)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 000f6b49  /system/lib/libart.so (_ZN3art17ConditionVariable4WaitEPNS_6ThreadE+96)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 002bec89  /system/lib/libart.so (_ZN3art7Monitor4WaitEPNS_6ThreadExibNS_11ThreadStateE+1144)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 002bf9e7  /system/lib/libart.so (_ZN3art7Monitor4WaitEPNS_6ThreadEPNS_6mirror6ObjectExibNS_11ThreadStateE+142)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 002d123b  /system/lib/libart.so (_ZN3artL11Object_waitEP7_JNIEnvP8_jobject+38)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 0000037f  /data/dalvik-cache/arm/system@framework@boot.oat (Java_java_lang_Object_wait__+74)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Object.wait!(Native method)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - waiting on <0x0209961c> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Daemons$ReferenceQueueDaemon.run(Daemons.java:147)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - locked <0x0209961c> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Thread.run(Thread.java:818)
05-11 10:53:32.191 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "FinalizerDaemon" prio=5 tid=5 Waiting
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d5b6a0 self=0xacc79500
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22448 nice=0 cgrp=default sched=0/0 handle=0xb416a930
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 559427 30312 10 ) utm=0 stm=0 core=0 HZ=100
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb4068000-0xb406a000 stackSize=1038KB
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 000175e4  /system/lib/libc.so (syscall+28)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 000f6b49  /system/lib/libart.so (_ZN3art17ConditionVariable4WaitEPNS_6ThreadE+96)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 002bec89  /system/lib/libart.so (_ZN3art7Monitor4WaitEPNS_6ThreadExibNS_11ThreadStateE+1144)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 002bf9e7  /system/lib/libart.so (_ZN3art7Monitor4WaitEPNS_6ThreadEPNS_6mirror6ObjectExibNS_11ThreadStateE+142)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 002d1275  /system/lib/libart.so (_ZN3artL13Object_waitJIEP7_JNIEnvP8_jobjectxi+44)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 0000056d  /data/dalvik-cache/arm/system@framework@boot.oat (Java_java_lang_Object_wait__JI+96)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Object.wait!(Native method)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - waiting on <0x0f79a225> (a java.lang.ref.ReferenceQueue)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Object.wait(Object.java:423)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:101)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - locked <0x0f79a225> (a java.lang.ref.ReferenceQueue)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:72)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Daemons$FinalizerDaemon.run(Daemons.java:185)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Thread.run(Thread.java:818)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "FinalizerWatchdogDaemon" prio=5 tid=6 Waiting
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d5b700 self=0xacc79a00
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22449 nice=0 cgrp=default sched=0/0 handle=0xb4063930
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 449844 0 4 ) utm=0 stm=0 core=3 HZ=100
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb3f61000-0xb3f63000 stackSize=1038KB
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 000175e4  /system/lib/libc.so (syscall+28)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 000f6b49  /system/lib/libart.so (_ZN3art17ConditionVariable4WaitEPNS_6ThreadE+96)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 002bec89  /system/lib/libart.so (_ZN3art7Monitor4WaitEPNS_6ThreadExibNS_11ThreadStateE+1144)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 002bf9e7  /system/lib/libart.so (_ZN3art7Monitor4WaitEPNS_6ThreadEPNS_6mirror6ObjectExibNS_11ThreadStateE+142)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 002d123b  /system/lib/libart.so (_ZN3artL11Object_waitEP7_JNIEnvP8_jobject+38)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 0000037f  /data/dalvik-cache/arm/system@framework@boot.oat (Java_java_lang_Object_wait__+74)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Object.wait!(Native method)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - waiting on <0x03cf7ffa> (a java.lang.Daemons$FinalizerWatchdogDaemon)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Daemons$FinalizerWatchdogDaemon.waitForObject(Daemons.java:255)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - locked <0x03cf7ffa> (a java.lang.Daemons$FinalizerWatchdogDaemon)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Daemons$FinalizerWatchdogDaemon.run(Daemons.java:227)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Thread.run(Thread.java:818)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "HeapTaskDaemon" prio=5 tid=7 Blocked
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d5b760 self=0xacc7a900
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22450 nice=0 cgrp=default sched=0/0 handle=0xb3f59930
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 483021 0 6 ) utm=0 stm=0 core=3 HZ=100
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb3e57000-0xb3e59000 stackSize=1038KB
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 000175e4  /system/lib/libc.so (syscall+28)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 000f6b49  /system/lib/libart.so (_ZN3art17ConditionVariable4WaitEPNS_6ThreadE+96)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 001d73f5  /system/lib/libart.so (_ZN3art2gc13TaskProcessor7GetTaskEPNS_6ThreadE+104)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 001d7907  /system/lib/libart.so (_ZN3art2gc13TaskProcessor11RunAllTasksEPNS_6ThreadE+38)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 0000037f  /data/dalvik-cache/arm/system@framework@boot.oat (Java_dalvik_system_VMRuntime_runHeapTasks__+74)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at dalvik.system.VMRuntime.runHeapTasks(Native method)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   - waiting to lock an unknown object
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Daemons$HeapTaskDaemon.run(Daemons.java:355)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   at java.lang.Thread.run(Thread.java:818)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "Binder_1" prio=5 tid=8 Native
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d670a0 self=0xaee53400
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22451 nice=0 cgrp=default sched=0/0 handle=0xb3d53930
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 2036614 1544584 11 ) utm=0 stm=0 core=0 HZ=100
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb3c57000-0xb3c59000 stackSize=1014KB
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 000175e4  /system/lib/libc.so (syscall+28)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 000f6b49  /system/lib/libart.so (_ZN3art17ConditionVariable4WaitEPNS_6ThreadE+96)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 00110cc3  /system/lib/libart.so (_ZN3art8CheckJNI11CallMethodVEPKcP7_JNIEnvP8_jobjectP7_jclassP10_jmethodIDSt9__va_listNS_9Primitive4TypeENS_10InvokeTypeE+870)
05-11 10:53:32.192 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 00112419  /system/lib/libart.so (_ZN3art8CheckJNI18CallBooleanMethodVEP7_JNIEnvP8_jobjectP10_jmethodIDSt9__va_list+36)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 000845f7  /system/lib/libandroid_runtime.so (???)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 000880af  /system/lib/libandroid_runtime.so (???)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #06 pc 000199b1  /system/lib/libbinder.so (_ZN7android7BBinder8transactEjRKNS_6ParcelEPS1_j+60)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #07 pc 0001ecc5  /system/lib/libbinder.so (_ZN7android14IPCThreadState14executeCommandEi+532)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #08 pc 0001ee31  /system/lib/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+64)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #09 pc 0001ee95  /system/lib/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+48)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #10 pc 000237e1  /system/lib/libbinder.so (???)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #11 pc 0001006d  /system/lib/libutils.so (_ZN7android6Thread11_threadLoopEPv+112)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #12 pc 0005ee23  /system/lib/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+70)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #13 pc 0003f45f  /system/lib/libc.so (_ZL15__pthread_startPv+30)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #14 pc 00019b43  /system/lib/libc.so (__start_thread+6)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   (no managed stack frames)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] "Binder_2" prio=5 tid=9 Native
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | group="" sCount=1 dsCount=0 obj=0x12d6a0a0 self=0xacc7ae00
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | sysTid=22452 nice=0 cgrp=default sched=0/0 handle=0xb3c4f930
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | state=S schedstat=( 1415885 679949 4 ) utm=0 stm=0 core=0 HZ=100
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | stack=0xb3b53000-0xb3b55000 stackSize=1014KB
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   | held mutexes=
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #00 pc 000409bc  /system/lib/libc.so (__ioctl+8)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #01 pc 00047439  /system/lib/libc.so (ioctl+14)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #02 pc 0001e909  /system/lib/libbinder.so (_ZN7android14IPCThreadState14talkWithDriverEb+132)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #03 pc 0001edf7  /system/lib/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+6)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #04 pc 0001ee95  /system/lib/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+48)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #05 pc 000237e1  /system/lib/libbinder.so (???)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #06 pc 0001006d  /system/lib/libutils.so (_ZN7android6Thread11_threadLoopEPv+112)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #07 pc 0005ee23  /system/lib/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+70)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #08 pc 0003f45f  /system/lib/libc.so (_ZL15__pthread_startPv+30)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   native: #09 pc 00019b43  /system/lib/libc.so (__start_thread+6)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366]   (no managed stack frames)
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/art: art/runtime/runtime.cc:366] 
                                                         
                                                         --------- beginning of crash
05-11 10:53:32.193 22440-22440/x.xx.samplejni A/libc: Fatal signal 6 (SIGABRT), code -6 in tid 22440 (xx.samplejni)
05-11 10:53:32.295 196-196/? A/DEBUG: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
05-11 10:53:32.295 196-196/? A/DEBUG: Build fingerprint: 'google/hammerhead/hammerhead:6.0.1/MOB30Y/3067468:user/release-keys'
05-11 10:53:32.295 196-196/? A/DEBUG: Revision: '11'
05-11 10:53:32.295 196-196/? A/DEBUG: ABI: 'arm'
05-11 10:53:32.295 196-196/? A/DEBUG: pid: 22440, tid: 22440, name: xx.samplejni  >>> x.xx.samplejni <<<
05-11 10:53:32.295 196-196/? A/DEBUG: signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
05-11 10:53:32.324 196-196/? A/DEBUG: Abort message: 'art/runtime/java_vm_ext.cc:410] JNI DETECTED ERROR IN APPLICATION: JNI FindClass called with pending exception java.lang.ClassNotFoundException: Didn't find class "net.sqlcipher.database.SQLiteDatabase" on path: DexPathList[[zip file "/data/app/x.xx.samplejni-1/base.apk"],nativeLibraryDirectories=[/data/app/x.xx.samplejni-1/lib/arm, /data/app/x.xx.samplejni-1/base.apk!/lib/armeabi, /vendor/lib, /system/lib]]'
05-11 10:53:32.325 196-196/? A/DEBUG:     r0 00000000  r1 000057a8  r2 00000006  r3 b6faeb7c
05-11 10:53:32.325 196-196/? A/DEBUG:     r4 b6faeb84  r5 b6faeb34  r6 00000000  r7 0000010c
05-11 10:53:32.325 196-196/? A/DEBUG:     r8 b4cbf378  r9 b4d6a800  sl 00000001  fp 00000001
05-11 10:53:32.325 196-196/? A/DEBUG:     ip 00000006  sp be8f2000  lr b6d1db61  pc b6d1ff50  cpsr 40070010
05-11 10:53:32.357 196-196/? A/DEBUG: backtrace:
05-11 10:53:32.357 196-196/? A/DEBUG:     #00 pc 00041f50  /system/lib/libc.so (tgkill+12)
05-11 10:53:32.358 196-196/? A/DEBUG:     #01 pc 0003fb5d  /system/lib/libc.so (pthread_kill+32)
05-11 10:53:32.358 196-196/? A/DEBUG:     #02 pc 0001c30f  /system/lib/libc.so (raise+10)
05-11 10:53:32.358 196-196/? A/DEBUG:     #03 pc 000194c1  /system/lib/libc.so (__libc_android_abort+34)
05-11 10:53:32.358 196-196/? A/DEBUG:     #04 pc 000174ac  /system/lib/libc.so (abort+4)
05-11 10:53:32.358 196-196/? A/DEBUG:     #05 pc 00333971  /system/lib/libart.so (_ZN3art7Runtime5AbortEv+228)
05-11 10:53:32.358 196-196/? A/DEBUG:     #06 pc 000f45fb  /system/lib/libart.so (_ZN3art10LogMessageD2Ev+2226)
05-11 10:53:32.358 196-196/? A/DEBUG:     #07 pc 0025aa4f  /system/lib/libart.so (_ZN3art9JavaVMExt8JniAbortEPKcS2_+1550)
05-11 10:53:32.358 196-196/? A/DEBUG:     #08 pc 0025adfd  /system/lib/libart.so (_ZN3art9JavaVMExt9JniAbortVEPKcS2_St9__va_list+64)
05-11 10:53:32.359 196-196/? A/DEBUG:     #09 pc 000fd1d1  /system/lib/libart.so (_ZN3art11ScopedCheck6AbortFEPKcz+32)
05-11 10:53:32.359 196-196/? A/DEBUG:     #10 pc 001022e5  /system/lib/libart.so (_ZN3art11ScopedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE.constprop.95+5072)
05-11 10:53:32.359 196-196/? A/DEBUG:     #11 pc 00104493  /system/lib/libart.so (_ZN3art8CheckJNI9FindClassEP7_JNIEnvPKc+386)
05-11 10:53:32.359 196-196/? A/DEBUG:     #12 pc 000e4f97  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so
05-11 10:53:32.359 196-196/? A/DEBUG:     #13 pc 000e572b  /data/app/x.xx.samplejni-1/lib/arm/libsqlcipher.so
05-11 10:53:32.359 196-196/? A/DEBUG:     #14 pc 0025b387  /system/lib/libart.so (_ZN3art9JavaVMExt17LoadNativeLibraryEP7_JNIEnvRKNSt3__112basic_stringIcNS3_11char_traitsIcEENS3_9allocatorIcEEEEP8_jobjectPS9_+1238)
05-11 10:53:32.359 196-196/? A/DEBUG:     #15 pc 002d13d7  /system/lib/libart.so (_ZN3artL18Runtime_nativeLoadEP7_JNIEnvP7_jclassP8_jstringP8_jobjectS5_+194)
05-11 10:53:32.359 196-196/? A/DEBUG:     #16 pc 71c76105  /data/dalvik-cache/arm/system@framework@boot.oat (offset 0x1ec9000)
05-11 10:53:32.691 196-196/? A/DEBUG: Tombstone written to: /data/tombstones/tombstone_06
05-11 10:53:32.691 196-196/? E/DEBUG: AM write failed: Broken pipe





```

**Comment by developernotes** [comment]
Hi @nikz-dev 

Are you utilizing ProGuard? If so, would you temporarily disable it and rebuild your application to see if that is the cause? Thanks!

**Comment by Janmejoy** [comment]
Hi @developernotes 

I am getting the same issue using ProGuard. Its working fine without ProGuard. How can we can fix the above issue using ProGuard.

**Comment by developernotes** [comment]
Hi @Janmejoy 

[Here](https://raw.githubusercontent.com/sqlcipher/android-database-sqlcipher/v3.4.0/proguard-project.txt) is an older ProGuard sample file you might use as an example, although you may need to make some modification depending on the version of ProGuard you are using.

**Comment by Tapchicoma** [comment]
Hi @developernotes.

`Android Sqlcipher` can bundle required proguard rules into aar. They will be merged into main proguard file, when user will build application.

**Comment by nikz-dev** [comment]
I just add "sqlcipher.jar" in project & my problem is solved. Thanks for your suggestions.

**Comment by developernotes** [comment]
Hi @Tapchicoma 

That is certainly something we can consider for a future release. Thanks!

**Comment by developernotes** [comment]
Hi @nikz-dev 

We are happy to hear everything is working well for you now. Take care!
